boxoffice/forms/item.py

```diff
@@ -21,8 +65,8 @@ class ItemForm(forms.Form):
         validators=[forms.validators.DataRequired(__("Please select a category"))])
     quantity_total = forms.IntegerField(__("Quantity available"),
         validators=[forms.validators.DataRequired(__("Please specify the quantity available for sale"))])
-    # FIXME
-    # assignee_details = forms.TextAreaField(__("Assignee details"))
+    assignee_details = forms.TextAreaField(__("Assignee details"), filters=[format_json],
+        validators=[validate_json], default=ASSIGNEE_DETAILS_PLACEHOLDER)
```

Comment:
We have a FormGenerator feature in Baseframe that was meant for this. It's never been used in production.